agent/pkg/proxy/proxy.go

```diff
@@ -120,15 +119,7 @@ func (proxy *EdgeProxy) newProtocolFromSock(ip string, port int, conn net.Conn)
 	name := svcNameSets[1]
 
 	switch protoName {
-	case "http":
-		proto = &http.HTTP{
-			Conn:         conn,
-			SvcName:      name,
-			SvcNamespace: namespace,
-			Port:         port,
-		}
-		err = nil
-	case "tcp":
+	case "tcp", "http":
```

Comment:
better to add some comment here for "http" fallback to "tcp".

Reply:
yes, it need to add a comment